x/bank/invariants.go

```diff
@@ -6,11 +6,12 @@ import (
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/cosmos/cosmos-sdk/x/bank/types"
 )
 
 // register bank invariants
 func RegisterInvariants(ir sdk.InvariantRouter, ak auth.AccountKeeper) {
-	ir.RegisterRoute(RouterKey, "nonnegative-outstanding",
+	ir.RegisterRoute(types.RouterKey, "nonnegative-outstanding",
```

Comment:
I think this should be (from what I've seen in other modules):
```suggestion
	ir.RegisterRoute(types.ModuleName, "nonnegative-outstanding",
```

Reply:
~~also, why is the other invariant (`TotalCoinsInvariant`) not registered ? ~~

Actually think we can delete it as we'll be using the total supply invariant from `x/supply` instead

Reply:
renamed, let's remove it once total supply is merged in